packages/@n8n/task-runner/src/task-runner.ts

```diff
@@ -287,6 +305,8 @@ export abstract class TaskRunner {
 			data,
 		});
 		this.runningTasks.delete(taskId);
+		console.log(`Runner completed task ${taskId}`);
+		this.resetIdleTimer();
```

Comment:
Shouldn't we also reset it in case the task fails? I.e. move this to the end of `receivedSettings` method